### PR TITLE
chore: Adds React

### PR DESCRIPTION
**Pull request by tribakzero**
This PR adds a React project and the most awesome logo to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 THIS IS THE README HELLO
+HELLO README, THIS IS DAD.
```
Comment (UpsetPelican) [inline README.md]: `>:v`

Comment (tribakzero) [reply README.md]: :'c

File: src/src/App.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (UpsetPelican) [inline src/src/App.js]: OHSHIT Metes HTML con funciones de JS
https://media.giphy.com/media/Wq7xqylQsSVhK/giphy.gif

File: src/src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (UpsetPelican) [inline src/src/index.js]: no clue what this file does

File: src/src/App.js (hunk 2 above)
Comment (tribakzero) [reply src/src/App.js]: JSX Babe!

File: src/src/index.js (hunk 3 above)
Comment (tribakzero) [reply src/src/index.js]: HUR DUR, it's the main entrypoint, in some cases it has auth/error/logging wrappers on it.

**Pull request merged**